package.json

```diff
@@ -24,8 +24,10 @@
     "yarn": ">= 1.0.0"
   },
   "scripts": {
-    "start": "node server/app.js",
-    "test": "grunt test",
+    "start": "webpack-serve",
```

Comment:
You still have in `scripts` part `"update-webdriver"` which is using `grunt-protractor-runner`.

Should be completely removed right?

Reply:
Yes